triggers/incident-triggered/event.schema.json

```diff
@@ -0,0 +1,114 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "id": {
+      "type": "string",
+      "description": "The incident's unique identifier"
+    },
+    "incidentNumber": {
+      "type": "integer",
```

Comment:
This was changed from a string in the readme. This intentional?

Reply:
It is indeed an integer in their API response. Must have been a typo in the README.